src/hakunapi-core/src/main/java/fi/nls/hakunapi/core/telemetry/TelemetrySpan.java

```diff
@@ -0,0 +1,23 @@
+package fi.nls.hakunapi.core.telemetry;
+
+import fi.nls.hakunapi.core.request.WriteReport;
+
+public interface TelemetrySpan extends AutoCloseable {
+
+    default void counts(int count) {};
```

Comment:
Not sure if the default implementation should be a NOP. I would've marked these as required to implement and only implemented these as empty functions in the NOP implementation.

Reply:
I'll remove default method declarations

Reply:
fixed